### DISCUSSION TITLE
zotero 55版本以上chartero提取图片功能

### DISCUSSION THREAD

**Discussion by maydayashin143** [volatile-static/Chartero#93] [Q&A] [answered]
zotero 55版本以上chartero提取图片功能没有了，很喜欢这个功能，什么时候添加，之前版本有的时候会出错，在左栏注释框出现多个提取图片的窗口

**Comment by volatile-static** [comment, marked as answer]
添加这个功能需要官方提供API，这可能需要等很久。你可以暂时用这个插件代替：https://github.com/MuiseDestiny/zotero-figure

**Reply by maydayashin143** [reply]
好的，谢谢那等您的更新，figure提取图片在注释框或保存笔记里都没有chartero的功能好，等待，感谢
